plugins/nestjs-nats/package.json

```diff
@@ -11,7 +11,7 @@
     "license": "Apache-2.0",
     "devDependencies": {
         "@amplication/code-gen-types": "^1.1.19",
-        "@amplication/code-gen-utils": "^0.0.5-beta.0",
```

Comment:
Did you intentionally take it out of beta?

Reply:
Please pull the changes to your local clone. We use the `^0.0.5` I changed that after we publish the version